<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDMC Fae Diag Kit v0.4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QListWidget" name="functionList">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>ADB模式</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::NetworkWireless"/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>串口通信</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::SyncSynchronizing"/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>日志分析</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="functionTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageLogDiag">
       <layout class="QVBoxLayout" name="pageLogDiagLayout" stretch="0,0,1,0,0">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>视讯产品诊断工具</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="btnStartLog">
            <property name="text">
             <string>开始实时抓取日志</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStopLog">
            <property name="text">
             <string>停止日志抓取</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExportLog">
            <property name="text">
             <string>导出当前筛选日志</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnScreenshot">
            <property name="text">
             <string>设备截图</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="filterLayout">
          <item>
           <widget class="QLabel" name="filterKeywordLabel">
            <property name="maximumSize">
             <size>
              <width>109</width>
              <height>257</height>
             </size>
            </property>
            <property name="text">
             <string>日志过滤关键字:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filterKeywordEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="filterLevelLabel">
            <property name="text">
             <string>日志级别:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="filterLevelCombo"/>
          </item>
          <item>
           <widget class="QCheckBox" name="autoScrollCheck">
            <property name="text">
             <string>自动滚动</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="logTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="deviceInfoLayout">
          <item>
           <layout class="QVBoxLayout" name="deviceInfoTextLayout">
            <item>
             <widget class="QLabel" name="statusLabel">
              <property name="text">
               <string>设备状态: 未检测</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="serialLabel">
              <property name="text">
               <string>序列号: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="brandLabel">
              <property name="text">
               <string>品牌: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="modelLabel">
              <property name="text">
               <string>型号: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="androidLabel">
              <property name="text">
               <string>安卓版本: -</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="deviceImageLabel">
            <property name="text">
             <string>(无产品图片)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSerial">
       <layout class="QVBoxLayout" name="pageSerialLayout">
        <item>
         <layout class="QHBoxLayout" name="serialPortConfigLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelSerialPort">
            <property name="text">
             <string>串口:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="serialPortCombo"/>
          </item>
          <item>
           <widget class="QLabel" name="labelBaudRate">
            <property name="text">
             <string>波特率:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baudRateCombo">
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshPortsBtn">
            <property name="text">
             <string>刷新串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openPortBtn">
            <property name="text">
             <string>打开串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closePortBtn">
            <property name="text">
             <string>关闭串口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="serialLogTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageFunction3">
       <layout class="QVBoxLayout" name="pageFunction3Layout">
        <item>
         <widget class="QLabel" name="labelFunc3">
          <property name="text">
           <string>功能占位框</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
